.github/workflows/tests.yml

```diff
@@ -15,6 +15,10 @@ jobs:
       matrix:
         os: [ubuntu-latest, macos-latest, windows-latest]
         resolver: [nightly, lts-18, lts-16, lts-14]
+        includes:
+          - os: ubuntu-latest
+            resolver: nightly
+            extra_args: --stack-yaml=stack-ghc-9.2.yaml
```

Comment:
Actually, on closer inspection, it looks like the builds are reporting using this for _all_ builds now. I'd recommend unifying this with `resolver` as something like:

```yaml
resolver:
- --resolver nightly
- --resolver lts-18
- ...
- --stack-yaml=stack-ghc-9.2.yaml
```

Reply:
Sorry, can you try it again? I used `includes` instead of `include` 🤦

Reply:
Weird... I'm not sure why it's not working. But i'll change it anyway